sops/3scale-scaling.md

```diff
@@ -0,0 +1,65 @@
+# 3scale scaling on Openshift 3.11, Integreatly 1.4.x
```

Comment:
Might be worth renaming this to something like "3Scale Scaling Guide". We can deal with managing Openshift 3.11 & 4 in future iterations